keanu-project/src/main/java/io/improbable/keanu/algorithms/SampleStats.java

```diff
@@ -0,0 +1,73 @@
+package io.improbable.keanu.algorithms;
+
+import org.apache.commons.math3.complex.Complex;
+import org.apache.commons.math3.transform.DftNormalization;
+import org.apache.commons.math3.transform.FastFourierTransformer;
+import org.apache.commons.math3.transform.TransformType;
+
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+public final class SampleStats {
```

Comment:
Not a big deal, but I think for discoverability in Intellij it'd be better for the class name to have the word autocorrelation in it? And as we add more statistical tools, we could just put them all under the same namespace.

Reply:
And maybe some documentation or a relevant link for the class?

Reply:
That's a good idea. It could be `algorithms.Statistics.Autocorrelation`?
@migwellian, @GeorgeNash, what do you think?

Reply:
Decided to do this.